src/components/index.d.ts

```diff
@@ -336,21 +336,21 @@ export declare const Pagination: {
     /**
      * Ultimately, this returns the JSON.parse'd data
      */
-    fetchPageCallback: (pageNumber: number | string) => any
+    fetchPageCallback: (pageNumber: number | string) => object
```

Comment:
Relates to @gbockus feedback at https://github.com/getethos/ethos-design-system/pull/217#discussion_r362981617